Src/VsVim/VsVim.csproj

```diff
@@ -111,7 +113,7 @@
   <PropertyGroup>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.VSSDK.BuildTools" Version="15.5.72" />
+    <PackageReference Include="Microsoft.VSSDK.BuildTools" Version="15.9.3039" />

```

Comment:
The 15.5.72 version of `CreatePkgDef` had a bug when the package reference `Microsoft.VisualStudio.Threading` at version 14.0. There was an assembly binding redirect in the app.config file that forced it to 15.3. This failed as the version in the output directory of the VSIX would always be 14.0. The newer version removed this binding redirect and hence can work with 14.0 properly.